### DISCUSSION TITLE
Loras not working and also appear to not being loaded.

### DISCUSSION THREAD

**Discussion by DalekSkaro1** [vladmandic/sdnext#3303] [Q&A]
When trying to add a lora to the prompt the image generated does not seem to be affected by the lora, even with the same loras that I had used previously and that had then woked. When mentioning that the lora is loading on the command line, it does not appear to load ending with "Loading model: C:\AI2\models\Lora\add-detail-XL.safetensors -------------------------- 0.0/218 MB -:--:--" and "22:05:50-606574 INFO     LoRA apply: ['add-detail-XL'] patch=0.03 load=1.07" appearing in the prompt. This problem appears to have begun when the Olive options first appeared for me (I hadn't used SD in some time). I have tryed unistalling all of the automatic folder and re-installing it again. I am also sure that the loras aren't being applied because putting a weight of 500 does not generate a picture of noise but the same type of picture as if there wasn't any lora.

**Comment by vladmandic** [comment]
pls create an issue and fill are required info as based on this alone, there is not enough info to say much.
